src/categories/categories.js

```diff
@@ -0,0 +1,3 @@
+/* eslint-disable import/prefer-default-export */
+export const categories = ['Action', 'Biography', 'History',
+  'Horror', 'Kids', 'Learning', 'Sci-Fi'];
```

Comment:
Kindly define this constant with Books `categories`  inside your BooksForm.js and also to avoid Linters disabling